x-pack/docs/en/security/authentication/saml-guide.asciidoc

```diff
@@ -705,10 +713,12 @@ PUT /_security/role_mapping/saml-finance
   "enabled": true,
   "rules": { "all": [
         { "field": { "realm.name": "saml1" } },
-        { "field": { "groups": "finance-team" } }
+        { "field": { "groups": "finance-team" } } <1>
   ] }
 }
 --------------------------------------------------
+<1> The `groups` attribute supports using wildcards (`*`). Refer to the
+<<saml-es-user-properties,{es} user properties>> for more information.
```

Comment:
I think it would be better to link the [doc page for role mapping](https://elasticsearch_86770.docs-preview.app.elstc.co/guide/en/elasticsearch/reference/master/security-api-put-role-mapping.html#security-api-put-role-mapping-example) since it gives more examples of using not only wildcard, but also template. We don't currently document the usage of regex. But if we do, it will be part of the role mapping doc page as well.

Reply:
That change sounds good to me -- I'll update the link.